db/migrate/20180626125654_make_maintenance_zone_record.rb

```diff
@@ -0,0 +1,16 @@
+class MakeMaintenanceZoneRecord < ActiveRecord::Migration[5.0]
+  class Zone < ActiveRecord::Base
+  end
+
+  def up
+    say_with_time("Creating Maintenance Zone") do
+      Zone.create_with(:description => "Maintenance Zone", :visible => false).find_or_create_by!(:name => 'maintenance')
+    end
+  end
+
+  def down
+    say_with_time("Deleting Maintenance Zone") do
+      Zone.where(:name => 'maintenance').where(:visible => false).destroy_all
+    end
+  end
+end
```

Comment:
This entire migration is unnecessary, because this can be handled in Zone.seed.

Reply:
So do I have to add Zone.seed to every method which uses maintenance zone? 
I'm not sure how find_or_create in every method call is better than one data migration

Reply:
also this call should be ensured in Zone.maintenance_zone but I follower implementation of Zone.default_zone, where it is not. 
Rely on all developers, that they will call Zone.seed every time when they want to use one of these methods doesn't seem to be good idea

Reply:
@fryguy - are you sure that `Zone.seed` is called after migrations in all existing installations?